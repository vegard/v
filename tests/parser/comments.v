# Comment at the start.

x := foo(); # Comment at the end of a line.

y := # Comment in the middle of an expression.
	foo();

# Comment at the end.
