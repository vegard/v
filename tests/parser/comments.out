(semicolon
    (juxtapose
        (symbol_name _define)
        (juxtapose
            (symbol_name x)
            (juxtapose
                (symbol_name foo)
                (parens)
            )
        )
    )
    (juxtapose
        (symbol_name _define)
        (juxtapose
            (symbol_name y)
            (juxtapose
                (symbol_name foo)
                (parens)
            )
        )
    )
)
